apps/daimo-mobile/src/logic/trpc.tsx

```diff
@@ -59,11 +59,23 @@ function getOpts(daimoChain: DaimoChain) {
           init.headers["x-daimo-platform"] = platform;
           init.headers["x-daimo-version"] = version;
 
-          return fetch(input, init).then((res) => {
+          // Fetch timeout: 5 seconds
+          const timeout = 5000;
```

Comment:
might be too low for some edge cases: https://ui.honeycomb.io/daimo/environments/prod/datasets/daimo-api/result/qyMZ51U5haA?tab=events has P999 of 10-12k ms in a few spikes

Reply:
increased to 10s

outliers should be fixed by retrying